docs/source/index.rst

```diff
@@ -0,0 +1,47 @@
+.. Hyperledger Cactus documentation master file, created by
+   sphinx-quickstart on Thu Mar 31 08:42:27 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+.. automodule: ../../README.md
+   :members:
+
+Welcome to Hyperledger Cactus's documentation!
```

Comment:
Typo (I think): if there's an `s` at the end of the word then the second `s` needs to be omitted e.g.: `Cactus' documentation`.